docs/api-reference/automatic-compaction-api.md

```diff
@@ -258,6 +258,88 @@ Host: http://ROUTER_IP:ROUTER_PORT
 
 A successful request returns an HTTP `200 OK` message code and an empty response body.
 
+### Update cluster-level compaction config
+
+Updates cluster-level configuration for compaction tasks which applies to all datasources, unless explicitly overridden in the datasource compaction config.
+This includes the following fields:
+
+|Config|Description|Default value|
+|------|-----------|-------------|
+|`compactionTaskSlotRatio`|Ratio of number of slots taken up by compaction tasks to the number of total task slots across all workers.|0.1|
+|`maxCompactionTaskSlots`|Maximum number of task slots that can be taken up by compaction tasks.|2147483647 (i.e. total task slots)|
+|`compactionPolicy`|Policy to prioritize intervals for compaction|`newestSegmentFirst`|
+|`useSupervisors`|Whether compaction should be run on Overlord using supervisors instead of Coordinator duties.|false|
+|`engine`|Engine to use for running compaction tasks, native or MSQ.|Native|
```

Comment:
This should list the values exactly as you feed them into the API like `"native"` or `"msq"`. Does this property only apply if `useSupervisors: true`? if so it should be noted.